StreamChatClient_v3/WebSocketClient/WebSocketClient.swift

```diff
@@ -56,13 +56,13 @@ class WebSocketClient {
     private let eventDecoder: AnyEventDecoder
     
     /// The web socket engine used to make the actual WS connection
-    lazy var engine: WebSocketEngine = createEngine()
+    private(set) lazy var engine: WebSocketEngine = createEngine()
     
     /// If in the `waitingForReconnect` state, this variable contains the reconnection timer.
     private var reconnectionTimer: TimerControl?
     
     /// The queue on which web socket engine methods are called
-    private let engineQueue: DispatchQueue = .init(label: "io.getStream.chat.core.web_socket_engine_queue", qos: .default)
+    private let engineQueue: DispatchQueue = .init(label: "io.getStream.chat.core.web_socket_engine_queue", qos: .userInitiated)
```

Comment:
👍

I only suggest to rename it to `io.getstream.chat.core.web_socket_engine_queue`

Reply:
:) Let's do it next time we touch this file